app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_split = command.split()
+    old_file = command_split[1]
+    new_file = command_split[2]
```

Comment:
Prefer using unpacking to retrieve values from the command_split, e.g.:
```suggestion
    command_keyword, old_file_name, new_file_name = command_split
```
It will help to avoid retrieving values using non-descriptive indexing.